### PR TITLE
feat: FSE platform

### PR DESCRIPTION
**Pull request by ruphy**
temporary PR for FSE work

### CODE REVIEW

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>There is a problem with the commit message
> aggiornamenti FSE
- subject may not be empty
- type may not be empty</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 248f8b31b274ba5b6ec44bbce6ef74995376500e
</p>

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>There is a problem with the commit message
> aggiornamenti FSE
- subject may not be empty
- type may not be empty</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d439a5eb959ec9a306816afff08077e19132361c
</p>

File: _platforms/it/fse.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+## Aiutaci a migliorare
+
+Segnalaci le proposte di miglioramento e/o le eventuali anomalie che hai riscontrato nell’utilizzo della piattaforma attraverso [il canale Issue tracker](https://github.com/ministero-salute/it-fse-support/issues).
+
+## Canali
+
+<a class="btn btn-primary" href="https://github.com/ministero-salute/it-fse-support/issues" target="_blank"><i class="it-horn" /> Issue tracker</a>
+<a class="btn btn-primary" href="https://forum.italia.it" target="_blank"><i class="it-horn" /> Entra nel forum</a>
+<a class="btn btn-primary" href="https://developersitalia.slack.com/archives/C030BJEGEN8" target="_blank"><i class="it-comment" /> Canale Slack sviluppatori (#FSE)</a>
```
Comment (ruphy) [inline _platforms/it/fse.md:111]:
```suggestion
<a class="btn btn-primary" href="https://developersitalia.slack.com/archives/C03RDT88FSM" target="_blank"><i class="it-comment" /> Canale Slack sviluppatori (#FSE)</a>
```

File: _platforms/it/fse.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+comingsoon: false
+abilitante: false
+layout: platform
+hide_activities: true
+owners:
+  - name: Ministero della Salute
+    url: https://www.salute.gov.it/
+resources:
+  - Supporto Sviluppo:
+    - title: Pagina di supporto github
```
Comment (ruphy) [inline _platforms/it/fse.md:21]:
```suggestion
    - title: Repository su GitHub con materiale di supporto
```

**Comment by micoccis** [comment]
accettati commenti

File: _platforms/it/fse.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+ Il documento [Piattaforma di telemedicina ed Ecosistema FSE](https://github.com/ministero-salute/it-fse-support/tree/main/doc/telemedicina) fornisce maggiori dettagli su come il Gateway FSE e l’Ecosistema Dati Sanitari interoperano con le piattaforme di Telemedicina.
+
+
+## Codice della piattaforma
+
+Il codice sorgente del Gateway e dell’EDS (Ecosistema Dati Sanitari) verrà rilasciato con licenza open source raggiunta una prima release stabile. Il codice sarà pubblicato dal Ministero della Salute non appena disponibile.
+
+## Aiutaci a migliorare
+
+Segnalaci le proposte di miglioramento e/o le eventuali anomalie che hai riscontrato nell’utilizzo della piattaforma attraverso il [il sistema di issue su GitHub](https://github.com/ministero-salute/it-fse-support/issues) o il [canale slack #fse](https://developersitalia.slack.com/archives/C03RDT88FSM) su Developers Italia.
```
Comment (bfabio) [inline _platforms/it/fse.md:104]:
```suggestion
Segnalaci le proposte di miglioramento e/o le eventuali anomalie che hai riscontrato nell’utilizzo della piattaforma attraverso [il sistema di issue su GitHub](https://github.com/ministero-salute/it-fse-support/issues) o il [canale slack #fse](https://developersitalia.slack.com/archives/C03RDT88FSM) su Developers Italia.
```

File: _platforms/it/fse.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+## Prima fase di sperimentazione
+
+E' prevista una prima fase di sperimentazione che coinvolge un numero progressivo di regioni che possono candidarsi in raccordo con il Gruppo di coordinamento del tavolo FSE. Le regioni inviano 2 CSR, una per il certificato di autenticazione e una per il certificato di firma, alla casella di progetto [fse_support@sogei.it](mailto:fse_support@sogei.it), verranno emessi da Sogei i certificati di autenticazione e firma che consentiranno  l'accesso ai servizi della piattaforma di pre produzione che progressivamente verranno realizzati.
```
Comment (bfabio) [inline _platforms/it/fse.md:108]:
```suggestion
È prevista una prima fase di sperimentazione che coinvolge un numero progressivo di regioni che possono candidarsi in raccordo con il Gruppo di coordinamento del tavolo FSE. Le regioni inviano 2 CSR, una per il certificato di autenticazione e una per il certificato di firma, alla casella di progetto [fse_support@sogei.it](mailto:fse_support@sogei.it), verranno emessi da Sogei i certificati di autenticazione e firma che consentiranno  l'accesso ai servizi della piattaforma di pre produzione che progressivamente verranno realizzati.
```

**Review by bfabio — APPROVED** [review]

**Pull request merged**